components/layout-wrapper.tsx

```diff
@@ -41,30 +41,54 @@ const LayoutWrapper = ({ children }: Props) => {
             </Link>
             <ul className="relative hidden pt-8 md:flex">
               {leftHeaderNavLinks.map((link) => (
-                <li key={link.title}>
+                <>
+                  <li key={link.title}>
+                    <Link
+                      key={link.title}
+                      href={link.href}
+                      className="px-1.5 lg:px-2 xl:px-3 font-medium transition-colors duration-200 hover:text-gray-600"
+                    >
+                      <span>{link.title}</span>
+                      {link.href.startsWith('http') && (
+                        <span className="pl-0.5 text-gray-300">&#8599;</span>
+                      )}
+                    </Link>
+                  </li>
+                </>
+              ))}
+              {noSearch && (
+                <li className="px-1.5 lg:px-2 xl:px-3 font-medium transition-colors duration-200 hover:text-gray-600">
+                  <Link href="/search">Search</Link>
+                </li>
+              )}
+            </ul>
+            {noSearch == false && (
```

Comment:
Rather than `noSearch &&` and `noSearch == false`, normally in React you'd express this with a ternary:

```
{noSearch ? (
  <>Option A</>
) : (
  <>Option B</>
)}
```

I'd recommend doing that here.